app/models/snippet.rb

```diff
@@ -1,6 +1,8 @@
 class Snippet < ApplicationRecord
-  belongs_to :category
+  CATEGORIES = ["Ruby", "JavaScript", "CSS"]
+
   has_many :snippet_tags
   has_many :tags, through: :snippet_tags
   validates :title, presence: true
+  validates_inclusion_of :category, in: CATEGORIES
```

Comment:
[`validates_inclusion_of`](https://apidock.com/rails/ActiveModel/Validations/ClassMethods/validates_inclusion_of) is deprecated, let's use the [`inclusion`](https://guides.rubyonrails.org/active_record_validations.html#inclusion) option 👇 

```ruby
validates :category, inclusion: { in: CATEGORIES }
```